bcrypt.js

```diff
@@ -9,31 +9,43 @@ var crypto = require('crypto');
 
 /// generate a salt (sync)
 /// @param {Number} [rounds] number of rounds (default 10)
+/// @param {Buffer} [seed] random seed (default crypto.randomBytes(16))
```

Comment:
“Random seed” doesn’t seem like a great description for this. It’s an unencoded salt.